app/main.py

```diff
@@ -1,8 +1,50 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
The best approach is to move these parameters on the next lines after function name